Assets/MixedRealityToolkit.Tests/PlayModeTests/BoundsControlTests.cs

```diff
@@ -0,0 +1,314 @@
+
+            // No change to scale yet
+            Assert.AreEqual(initialScale, bbox.transform.localScale);
+
+            // Move hands beyond max scale limit
+            yield return hand.MoveTo(new Vector3(scaleHandler.ScaleMaximum * 2, scaleHandler.ScaleMaximum * 2, 0) + frontRightCornerPos, numHandSteps);
+
+            // Assert scale at max
+            Assert.AreEqual(Vector3.one * scaleHandler.ScaleMaximum, bbox.transform.localScale);
+
+            // Move hands beyond min scale limit
+            yield return hand.MoveTo(new Vector3(-scaleHandler.ScaleMinimum * 2, -scaleHandler.ScaleMinimum * 2, 0) + frontRightCornerPos, numHandSteps);
+
+            // Assert scale at min
+            Assert.AreEqual(Vector3.one * scaleHandler.ScaleMinimum, bbox.transform.localScale);
+
+            GameObject.Destroy(bbox.gameObject);
+            // Wait for a frame to give Unity a change to actually destroy the object
+            yield return null;
+
+        }
+
+        /// <summary>
+        /// Uses far interaction (HoloLens 1 style) to scale the bounds control
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator ScaleViaHoloLens1Interaction()
+        {
+            var bbox = InstantiateSceneAndDefaultBbox();
+            yield return null;
+            var bounds = bbox.GetComponent<BoxCollider>().bounds;
+            var startCenter = new Vector3(0, 0, 1.5f);
+            var startSize = new Vector3(.5f, .5f, .5f);
+            TestUtilities.AssertAboutEqual(bounds.center, startCenter, "bbox incorrect center at start");
+            TestUtilities.AssertAboutEqual(bounds.size, startSize, "bbox incorrect size at start");
+
+            PlayModeTestUtilities.PushHandSimulationProfile();
+            PlayModeTestUtilities.SetHandSimulationMode(HandSimulationMode.Gestures);
+
+            CameraCache.Main.transform.LookAt(bbox.ScaleHandles.Handles[3].transform);
+
+            var startHandPos = CameraCache.Main.transform.TransformPoint(new Vector3( 0.1f, 0f, 1.5f));
+            TestHand rightHand = new TestHand(Handedness.Right);
+            yield return rightHand.Show(startHandPos);
+            yield return rightHand.SetGesture(ArticulatedHandPose.GestureId.Pinch);
+            // After pinching, center should remain the same
+            var afterPinchbounds = bbox.GetComponent<BoxCollider>().bounds;
+            TestUtilities.AssertAboutEqual(afterPinchbounds.center, startCenter, "bbox incorrect center after pinch");
+            TestUtilities.AssertAboutEqual(afterPinchbounds.size, startSize, "bbox incorrect size after pinch");
+
+            var delta = new Vector3(0.1f, 0.1f, 0f);
+            yield return rightHand.Move(delta);
+
+            var endBounds = bbox.GetComponent<BoxCollider>().bounds;
+            TestUtilities.AssertAboutEqual(endBounds.center, new Vector3(0.033f, 0.033f, 1.467f), "endBounds incorrect center");
```

Comment:
These numerical values seem like they should be saved as constants?

Reply:
they're very specific to the tests - i saved the start size / center values but i'd rather keep those local - will put them in local vars